Core/src/org/sleuthkit/autopsy/modules/stix/STIXReportModule.java

```diff
@@ -108,34 +109,20 @@ public void generateReport(String baseReportDir, ReportProgressPanel progressPan
         reportAllResults = configPanel.getShowAllResults();
 
         // Set up the output file
-        try {
-            File file = new File(reportPath);
-            output = new BufferedWriter(new FileWriter(file));
-        } catch (IOException ex) {
-            logger.log(Level.SEVERE, String.format("Unable to open STIX report file %s", reportPath), ex); //NON-NLS
-            MessageNotifyUtil.Notify.show("STIXReportModule", //NON-NLS
-                    NbBundle.getMessage(this.getClass(),
-                            "STIXReportModule.notifyMsg.unableToOpenReportFile",
-                            reportPath),
-                    MessageNotifyUtil.MessageType.ERROR);
-            progressPanel.complete(ReportStatus.ERROR);
-            progressPanel.updateStatusLabel(
-                    NbBundle.getMessage(this.getClass(), "STIXReportModule.progress.completedWithErrors"));
-            return;
-        }
-
         // Keep track of whether any errors occur during processing
         boolean hadErrors = false;
 
         // Process the file/directory name entry
         String stixFileName = configPanel.getStixFile();
+        File reportFile = new File(reportPath);
```

Comment:
It would be better for "locality of reference" to move the creation of reportFile closer to where it is used.